src/Resources/config/services.xml

```diff
@@ -19,5 +19,12 @@
             <tag name="kernel.event_listener" event="kernel.controller" method="onKernelController" priority="-1" />
         </service>
         <service id="apy_breadcrumb_trail.annotation.listener" alias="APY\BreadcrumbTrailBundle\EventListener\BreadcrumbListener" public="true" />
+
+        <service id="Doctrine\Common\Annotations\AnnotationReader" />
```

Comment:
Doctrine annotations is optional (part of `require-dev`), so can not be mentioned as service for this bundle, as that will lead to exceptions on non-dev / prod runtime. 

Also, as autowiring on this bundle is enabled by default, there should not be a need to define this service from another dependency.